indexd/index/blueprint.py

```diff
@@ -264,6 +266,57 @@ def post_index_record():
     return flask.jsonify(ret), 200
 
 
+@blueprint.route('/index/blank/', methods=['POST'])
+@authorize
+def post_index_blank_record():
+    '''
+    Create a blank new record with only uploader field is filled
+    '''
+
+    uploader = flask.request.json.get('uploader')
+    if not uploader:
+        raise UserError('no uploader specified')
+
+    did, rev, baseid = blueprint.index_driver.add_blank_record(
+        uploader=uploader
+    )
+
+    ret = {
+        'did': did,
+        'rev': rev,
+        'baseid': baseid,
+    }
+
+    return flask.jsonify(ret), 200
```

Comment:
I think a 201 status code (`CREATED`) would make the most sense, though the other indexd endpoints already are just 200, so if you want to leave this alone that's fine

Reply:
Good point!. Fixed in 7caea53ee872202df117db75459f21526c7b2db1